src/sentry/sentry_metrics/querying/data_v2/transformation.py

```diff
@@ -174,39 +191,35 @@ def transform(self) -> Mapping[str, Any]:
         intervals = _build_intervals(start, end, interval)
 
         # We build the translated groups given the intermediate groups.
-        translated_groups = []
-        for group_key, group_metrics in intermediate_groups.items():
-            translated_serieses: dict[str, Sequence[ResultValue]] = {}
-            translated_totals: dict[str, ResultValue] = {}
-            for metric_name, metric_values in group_metrics.items():
-                series = metric_values.series
-                total = metric_values.total
-
-                # We generate the full series by passing as default value the identity of the totals, which is the
-                # default value applied in the timeseries. This function already aligns the series by sorting it in
-                # ascending order so there is no need to have the series elements sorted beforehand.
-                translated_serieses[metric_name] = _generate_full_series(
-                    int(start.timestamp()), len(intervals), interval, series, get_identity(total)
+        translated_queries_groups = []
```

Comment:
nit:
```suggestion
        transformed_queries_groups = []
```
since the method name is transform